cmd/lakectl/cmd/root.go

```diff
@@ -41,7 +42,11 @@ lakectl is a CLI tool allowing exploration and manipulation of a lakeFS environm
 			return
 		}
 		if _, ok := cfgFileErr.(viper.ConfigFileNotFoundError); ok {
-			// specific message in case the file doesn't not found
+			if emptyCfgFile {
```

Comment:
can you check `cfgFile` empty?

Reply:
It's never empty, since we use the default value. 
But if that file isn't found, you can still try and run with the default values or one read from the env file, no?